tests/test_fairworkflow.py

```diff
@@ -251,17 +251,17 @@ def test_draw_with_graphviz_module_and_dependency(self, tmp_path, test_workflow)
         test_workflow.draw(filepath=str(tmp_path))
 
     @mock.patch.dict('sys.modules', {'graphviz': None})
-    def test_display_without_graphviz_module(self, test_workflow):
-        """Test display method without graphviz python module installed."""
+    def test_display_rdf_without_graphviz_module(self, test_workflow):
+        """Test display_rdf method without graphviz python module installed."""
         with pytest.raises(ImportError):
-            test_workflow.display(full_rdf=True)
+            test_workflow.display_rdf()
 
-    def test_display_with_graphviz_module_and_dependency(self, test_workflow):
+    def test_display_rdf_with_graphviz_module_and_dependency(self, test_workflow):
         """
-        Test display method with graphviz python module and graphviz software
+        Test display_rdf method with graphviz python module and graphviz software
         installed
         """
-        test_workflow.display(full_rdf=True)
+        test_workflow.display_rdf()
```

Comment:
Likewise, maybe we could skip these tests if the python version is `3.8.0` and above?

Reply:
I created a new issue for this: #182